pkg/controller/storagecluster/quickstarts.go

```diff
@@ -0,0 +1,32 @@
+package storagecluster
+
+import (
+	"context"
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	consolev1 "github.com/openshift/api/console/v1"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcileStorageCluster) ensureQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	for _, qs := range AllQuickStarts {
+		cqs := consolev1.ConsoleQuickStart{}
+		err := yaml.Unmarshal([]byte(qs), &cqs)
+		if err != nil {
+			reqLogger.Info("failed to unmarshal consolequickstarts")
+			return err
+		}
+		found := consolev1.ConsoleQuickStart{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: cqs.Name, Namespace: cqs.Namespace}, &found)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				reqLogger.Info("Creating Quickstarts")
+				return r.client.Create(context.TODO(), &cqs)
+			}
+			return err
+		}
```

Comment:
Don't we want to add an update condition here? Or, do we want to create and forget?

Reply:
i am not sure what exactly we need to update for if the quickstarts are already found. Can you elaborate any reason

Reply:
What if you want to change it in next release?